docassemble/AssemblyLine/al_document.py

```diff
@@ -908,7 +917,7 @@ def as_editable_list(self, key:str='final', refresh:bool=True) -> List[DAFile]:
   def download_list_html(self, key:str='final', format:str='pdf', view:bool=True,
       refresh:bool=True, include_zip:bool = True, view_label="View", view_icon:str="eye",
       download_label:str="Download", download_icon:str="download", zip_label:str=None,
-      zip_icon:str="file-archive") -> str:
+      zip_icon:str="file-archive", include_original_in_zip=True) -> str:
```

Comment:
This argument isn't ever used

Reply:
```suggestion
      zip_icon:str="file-archive") -> str:
```

Reply:
Yep, I was just fixing that, sorry.

Reply:
Nm, you caught it as the same time as I did.